templates/institutional/institution-landing-en.html

```diff
@@ -0,0 +1,186 @@
+---
+title: "[Institution-landing-page]"
+layout: no-container
+language: "en"
+altLangPage: "institution-landing-fr.html"
+pageclass: "page-type-nav"
+pageType: "institution-landing"
+breadcrumb: [
+	{ "title": "Canada.ca", "link": "https://www.canada.ca/en.html" },
+	{ "title": "Departments & agencies", "link": "https://www.canada.ca/en/government/dept.html" }
+]
+dateModified: "2021-05-17"
+share: true
+noMainContainer: true
+followUsLayout: "inline"
+---
+<div class="provisional gc-institutional-bg mrgn-tp-md"
+	data-bgimg="img/ip-cover-image-1200x726.jpg">
+	<div class="container">
+		<div class="row">
+			<div class="col-md-7">
+				<h1 property="name" id="wb-cont">[Institution name]</h1>
+				<p>Short description of the institution’s mandate.</p>
+				<a class="btn btn-call-to-action" href="#">Super task button [optional]</a>
+			</div>
+		</div>
+	</div>
+</div>
```

Comment:
This can be either an component utility "hero" feature or you can leverage a set of existing utility class to get the desired effect.
Note that the information/data contained inside this div, do not provide information related to "gc-institutional-bg". That information belong more to the page not the hero image.